packages/nextjs/package.json

```diff
@@ -2,41 +2,26 @@
   "name": "@t3-oss/env-nextjs",
   "version": "0.9.1",
   "type": "module",
-  "keywords": [
-    "create-t3-app",
-    "environment variables",
-    "zod",
-    "nextjs"
-  ],
+  "keywords": ["create-t3-app", "environment variables", "zod", "nextjs"],
   "author": "Julius Marminge",
   "license": "MIT",
   "repository": {
     "type": "git",
     "url": "https://github.com/t3-oss/t3-env",
     "directory": "packages/nextjs"
   },
-  "main": "dist/index.js",
```

Comment:
I believe this line is breaking the import. Typescript just can't find it anymore. It works on 0.9.1

Reply:
Any chance this was an wrong change, @juliusmarminge ?

Reply:
It's only breaking if your tsconfig cant read exports field

Reply:
same to me: 

My tsconfig:

```json
{
  "compilerOptions": {
    "tsBuildInfoFile": "node_modules/.cache/typescript/tsbuildinfo.json",
    "allowJs": true,
    "skipLibCheck": true,
    "strict": true,
    "forceConsistentCasingInFileNames": true,
    "noEmit": true,
    "target": "ES2022",
    "esModuleInterop": true,
    "module": "ESNext",
    "moduleResolution": "Node",
    "resolveJsonModule": true,
    "isolatedModules": true,
    "jsx": "preserve",
    "lib": ["ESNext", "DOM", "DOM.Iterable"],
    "noUncheckedIndexedAccess": true,
      "incremental": true,
    "baseUrl": ".",
    "paths": {
      "@/*": ["./*"]
    },
    "plugins": [
      {
        "name": "next"
      }
    ],
  },
  "include": [
    "next-env.d.ts",
    "next-auth.d.ts",
    "**/*.ts",
    "**/*.tsx",
    ".next/types/**/*.ts"
  ],
  "exclude": ["node_modules"]
}
```

Reply:
I had to change 
```json
    "module": "ESNext",
    "moduleResolution": "Node",
```
to
```json
    "module": "NodeNext",
    "moduleResolution": "NodeNext",
```

and it's working now

Reply:
I'm not seeing typings being locatable or available in `0.10.1` when installed. Additionally, npmjs.org doesn't report them as being available, either. See the screenshot of released versions:

![npmjs-typescript](https://github.com/t3-oss/t3-env/assets/3775/83be6f29-9cc8-4d33-aac0-56bceeae574c)

Reply:
You'll get types through exports condition if you have a modern moduleResolution

Reply:
@juliusmarminge : Ahh ... you're correct. Using something like `moduleResolution: "bundler"` correctly identifies the types. Thank you!